addons/stock/report/report_stock_forecasted.py

```diff
@@ -158,10 +158,15 @@ def _reconcile_out_with_ins(lines, out, ins, demand, only_matching_move_dest=Tru
         in_domain, out_domain = self._move_confirmed_domain(
             product_template_ids, product_variant_ids, wh_location_ids
         )
-        outs = self.env['stock.move'].search(out_domain, order='priority desc, date, id')
+        outs = self.env['stock.move'].search(out_domain, order='reservation_date, priority desc, date, id')
```

Comment:
Why not filter out partially and assigned moves (`('state', 'in', ('partially_available', 'assigned'))`) ?

Reply:
Well, I see. Then it is a bad idea to the filter (`('state', 'not in', ('partially_available', 'assigned'))`). 
Sorry my bad.